src/api/index.ts

```diff
@@ -31,14 +31,21 @@ export const getETHBalance = async (account?: Account) => {
 }
 
 // ETH token balance
-export const getCurrentBalance = async (account?: Account) => {
-  const { Tokens } = await promisedAPI
+export const getCurrentBalance = async (tokenName?: TokenCode | 'ETH', account?: Account) => {
+  account = await fillDefaultAccount(account)
+  
+  if (tokenName && tokenName === 'ETH') {
+    const { getETHBalance } = await promisedWeb3
+
+    return getETHBalance(account)
+  }
 
+  const { Tokens } = await promisedAPI
   // account would normally be taken from redux state and passed inside an action
   // but just in case
-  account = await fillDefaultAccount(account)
 
-  return Tokens.getTokenBalance('ETH', account)
+  // should probably change name here to WETH
+  return Tokens.getTokenBalance('W-ETH', account)
```

Comment:
what about using `tokenName` here, maybe defaulting it to `W-ETH`

Reply:
that was my initial idea, and it worked but I'm honestly having second thoughts about this W-ETH thing. Rn I'm on the same PR and reverting back a commit. I think I want to keep the ETH name the same and just run a conditional there to check balance different if `tokenName` === `ETH`